data_and_models/pipelines/ner/dvc.lock

```diff
@@ -139,20 +19,20 @@ stages:
       md5: 278779756800bc6d1ec980d48bd0dc8c
       size: 1715
     - path: eval.py
-      md5: 3c9e86f7cbfaea90929e96a6443a16d4
-      size: 3391
+      md5: 8b7c9e4c20e6b0dd7f7daa0097be8e1a
+      size: 3407
     params:
       params.yaml:
-        eval.DISEASE:
+        eval.disease:
           etype_name: DISEASE
```

Comment:
I see that entity types got consistently renamed as lowercase. Is there a reason for this?
Also notice that the `etype_name` just below is uppercase, which seems contrasting.

Reply:
Also, hijacking this thread to as a related question. Is there a reason why we write
```yaml
CHEMICAL:
  etype_name: CHEBI
```
rather than
```yaml
CHEMICAL: CHEBI
```
I know it was not introduced in this PR, I just never thought about it before.

Reply:
The main idea was to avoid duplication of `parameters` in the `dvc.yaml`.  Indeed, `params.yaml` keys are expecting uppercase names and `metrics` filenames are lowercase. I could not find a way (maybe there is one) to modify directly the strings in the `dvc.yaml` from one form to the other. From what @pafonta told me, `spaCy 3.0` allows to do this conversion of `entity types` directly in the model. So I thought it is better to change `params.yaml` that is going to change anyway in the future (see comment from [PA](https://github.com/BlueBrain/Search/pull/324#issuecomment-810058603)).

However, here are the different possibilities (in my opinion):
- Having 2 parameters in `dvc.yaml` for entity types (lower and uppercase), or finding a way to changing strings directly in the `dvc.yaml` if possible. (This solution does not affect the rest of the files)
- Changing the `params.yaml` keys to lowercase (as done here - affects `eval.py` and `params.yaml`)
- Going for metric filename that are uppercase instead of lowercase. 

What do you think about them @Stannislav @FrancescoCasalegno ? Do not hesitate if you have other ideas for this. 

Regarding the idea: 
```yaml
CHEMICAL: CHEBI
```
I maybe need to think about it to be sure there is no problem but it is definitely worth to check. Thanks for the suggestion!

Reply:
> From what @pafonta told me, spaCy 3.0 allows to do this conversion of entity types directly in the model.

The idea would be to rename the labels directly in the pipeline.

One could add an `EntityRuler` in `config.cfg` (related to #310) with a `ENT_TYPE` pattern for each label.

This way, `nlp(text).ents` would have our labels only and free us from managing them outside the `spaCy` pipeline.

Reply:
@EmilieDel since you're saying that `dvc.yaml` and `params.yaml` will change anyway, maybe the upper/lower case discussion is not so important.

Personally I like things spelled out in the most direct way (like I wrote above `CHEMICAL: CHEBI`), but this kind of restructuring might be out of scope for this PR.